tonic-desktop/README.md

```diff
@@ -1,4 +1,4 @@
-# A React example destop app built with Operator Framework
+# A React example destop app built with Socket SDK
 
 This app uses [op-node](https://github.com/socketsupply/op-node) as a backend and [Tonic](https://tonicframework.dev) code bundled with [esbuild](https://esbuild.github.io) for a frontend.
```

Comment:
`socket-node`

Reply:
that will be my next PR, then -base + 3 apps